FoodOrderingSystem.py

```diff
@@ -0,0 +1,884 @@
+            if employee_options == 1:
+                category_name = input("Enter category name: ")
+                fos.add_food_category(category_name)
+
+            elif employee_options == 2:
+                category_id = input ("Enter category ID: ")
+                food_name = input("Enter food name: ")
+                price = input("Food price: ")
+                fos.add_food_details(category_id, food_name, price)
+
+            elif employee_options == 3:
+                delivery_person_name = input("Enter delivery person name: ")
+                delivery_person_phone = input("Enter delivery person phone: ")
+                fos.add_delivery_person(delivery_person_name, delivery_person_phone)
+
+            elif employee_options == 4:
+                order_id = input("Enter order ID: ")
+                delivery_person_id = input("Enter delivery person ID: ")
+                fos.assign_deliver_person_to_deliver_order(order_id, delivery_person_id) 
+
+            elif employee_options == 5:
+                order_id = input("Enter order ID: ")
+                select = """                 
+                To update order, press
+                
+                1. En route
+                2. Delivered
+
+                Your option: 
+ 
+                """
+                
+                options = int(input(select))
+                if options == 1:
+                    order_status = "En route"
+                    fos.update_order(order_id, order_status) 
+
+                if options == 2:
+                    order_status = "Delivered"
+                    fos.update_order(order_id, order_status)
+
+            elif employee_options == 6:
+                order_id = input("Enter order ID: ")
+                fos.view_orders(order_id)
+                fos.view_order_grand_total(order_id)
+
+            elif employee_options == 7:
+                order_id = input("Enter order ID: ")
+                fos.view_order_status(order_id)
+                fos.view_orders(order_id)
+
+            elif employee_options == 8:
+                select = """ 
+                To see revenue for a certain status, please press 
+                
+                1. Checkedout
+                2. En route
+                3. Delivered
+                
+                Your option:                 
+
+                """                
+                options = int(input(select))
+                if options == 1:
+                    order_status = "Checkedout"
+                elif options == 2:
+                    order_status = "En route"
+                else:
+                    order_status = "Delivered"
+                fos.view_sales_today(order_status)
+                fos.sum_revenue_today(order_status)
+                
+            elif employee_options == 9:
+                order_id = input("Enter order ID: ")
+                fos.delete_order(order_id)
+
+            employee_options = int(input("\nYour option: "))
```

Comment:
The main function is still too big, break it up into `process_employee_options_flow()` and `process_customer_options_flow()`

Reply:
Changes made.

broke the main into 

def process_employee_options_flow()
def process_customer_options_flow()
def process_order_flow()